app/components/preprint-status-banner/component.js

```diff
@@ -229,7 +222,9 @@ export default Component.extend({
     }),
 
     init() {
-        this.get('submission.actions').then(this._handleActions.bind(this));
+        this.get('submission.actions')
+            .then(this._getRecentAction.bind(this))
```

Comment:
`_getRecentAction` seems unnecessary, could just be `.then(latestAction)`, I think.